src/Riok.Mapperly/Descriptors/Mappings/ExistingTarget/EnsureCapacityBuilder.cs

```diff
@@ -0,0 +1,131 @@
+﻿using System.Diagnostics.CodeAnalysis;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CSharp;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
+using Riok.Mapperly.Helpers;
+using static Microsoft.CodeAnalysis.CSharp.SyntaxFactory;
+using static Riok.Mapperly.Emit.SyntaxFactoryHelper;
+
+namespace Riok.Mapperly.Descriptors.Mappings.ExistingTarget;
+
+public class EnsureCapacityInfo
```

Comment:
This code base usually follows one type per file approach.

To match existing architecture convert these properties to private fields and create an exposed method `BuildEnsureCapacity()` which creates the syntax (content of `TryBuildEnsureCapacityStatement`) (similar to how the object factories and mappings work).